ydb/docs/en/core/cluster/toc_i.yaml

```diff
@@ -25,3 +21,15 @@ items:
   href: ../maintenance/manual/change_actorsystem_configs.md
 - name: Updating configurations via CMS
   href: ../maintenance/manual/cms.md
+- name: Audit logging
+  items:
+    - name: General info and setup
+      href: audit-log.md
+    - name: Schema operations
+      href: audit-log-scheme.md
+    - name: DML operations
+      href: audit-log-dml.md
```

Comment:
Two inconsistencies here:
1. "Schem**a**" in title, but "schem**e**" in URL
2. DDL and DML would've looked better next to each other